make test fails if inside a pipenv shell

* koji_wrapper version: master
* Python version: python 3.6.2
* Operating System: Linux

### Description

While trying to run the tests locally using `make test`, the command fails.

### Expected
Tests run and pass.

### What I did
```
$ pipenv shell
$ make test
pip3 install --user pipenv
Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
make: *** [Makefile:88: init] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -26,6 +26,8 @@ export PRINT_HELP_PYSCRIPT
 
 BROWSER := @pipenv run python -c "$$BROWSER_PYSCRIPT"
 
+PIPENV_AVAILABLE := $(shell command -v pipenv 2> /dev/null)
+
 help:
 	@pipenv run python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
 
@@ -85,7 +87,9 @@ dist: clean ## builds source and wheel package
 	ls -l dist
 
 init: ## install the dev environment
+ifndef PIPENV_AVAILABLE
 	pip3 install --user pipenv
+endif
 
 install: init ## install the package to the active Python's site-packages
 	pipenv run python setup.py install

